app/src/main/res/values/strings.xml

```diff
@@ -302,4 +302,27 @@
     <string name="dismiss_temporary">Not now</string>
     <string name="update_available">New update available</string>
     <string name="update_available_description">A new version (%s) is available for download.</string>
+
+    <string name="installer_result_failed">Failed to install</string>
+    <string name="installer_result_cancelled">Cancelled</string>
+    <string name="installer_result_blocked">Blocked</string>
+    <string name="installer_result_conflict">Conflict</string>
+    <string name="installer_result_incompatible">Incompatible</string>
+    <string name="installer_result_invalid">Invalid</string>
+    <string name="installer_result_storage_issue">Storage issue</string>
+    <string name="installer_result_timeout">Timeout</string>
+    <string name="installer_result_success">Success</string>
+    <string name="installer_result_unknown">Unknown</string>
+
+    <string name="installer_result_failed_description">The installation failed for an unknown reason.</string>
+    <string name="installer_result_aborted_description">The installation was manually cancelled.</string>
+    <string name="installer_result_blocked_description">The installation was blocked.</string>
+    <string name="installer_result_conflict_description">An existing installation of the app prevents the installation. Reinstall by uninstalling the app and try again?</string>
```

Comment:
```suggestion
    <string name="installer_result_conflict_description">An existing installation of the app prevents the installation. Uninstall the app and try again?</string>
```

`Reinstall by uninstalling the app and try[ing] again.` is slightly confusing, especially if the currently installed app is an unpatched app.

Reply:
In https://github.com/ReVanced/revanced-manager/pull/1473/files#r1388757664 you suggest `Uninstall the current installation and try again?` but here you suggest `Uninstall the app and try again?`. Please chose either.

Reply:
Here in the case of a conflict, the first sentence in the dialog gives context to what app should be uninstalled, so `Uninstall the app and try again?` is sufficient.

In the case of an invalid package, more clarification is needed due to the lack of context from the first sentence in that dialog. (This is also why I suggested the term "current installation" for that dialog instead of "existing installation", which is the term used in the conflict dialog.)

If you must use the same sentence for both, you can use `Uninstall the existing installation and try again?`